wait-for-idle-runner/main.js

```diff
@@ -0,0 +1,27 @@
+const { Octokit } = require("@octokit/action");
```

Comment:
Is there any JS linting we could do of this file?  If we can't run it it'd be good to have some sort of linting.

Reply:
I have no clue about js linting. I can ask in the maintainer channel.

Note that no other js script in this repo seems to have .js linting.

I am willing to do the extra step to add js linting to this repo to get this merged, but on the other side it feels like yet another bigger project, especially if I have to fix js style issues in 10 other actions: js is not my programming language at all so this might take some more time.

Reply:
> Note that no other js script in this repo seems to have .js linting.

That's because they are all actually run. As-is there's no CI here to provide guardrails to avoid breaking this script.

Reply:
I'm not really the right person to ask about javascript stuff but I do know that you can use `node --check` to check for syntax errors which is definitely better than nothing.

```console
$ cat test.js
console.log("hello, world!"

$ node --check test.js
/Users/rylanpolster/Development/homebrew/actions/wait-for-idle-runner/test.js:1
console.log("hello, world!"
            ^^^^^^^^^^^^^^^

SyntaxError: missing ) after argument list
    at Object.compileFunction (node:vm:355:18)
    at wrapSafe (node:internal/modules/cjs/loader:1039:15)
    at checkSyntax (node:internal/main/check_syntax:66:3)
    at node:internal/main/check_syntax:39:3
```

Reply:
If the language was Python or Java I would have mocked the API calls and/or created a --dry-run version, but as this is javascript I am not sure if I want to spend more time on this right now. I added the `node --check` test.